CryptKeeperZK/crypt-keeper-extension
# feat: group demo

## Explanation

This PR adds demonstartion for bandada group interaction.

Details are below:
- [x] Add join group demo
- [x] Add generate group merkle proof demo
- [x] Minor test fixes

## Related Issues

Related #316 
Blocked by #872 

## Screenshots

<img width="1436" src="https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/751bf574-c57a-4410-9a3f-698f5633fcb6">

## Manual Testing Steps

1. Go to demo
2. Create new identity and connect
3. Try to generate group merkle proof (see blocked flow)
4. Try to join test group
5. Try to join test group again (see blocked flow)
6. Try to generate group merkle proof again

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
